### ISSUE TITLE
Downloading precomputed meshes from sharded multi level mesh sources fails

### ISSUE THREAD

**Issue by Raphtor** [sdorkenw/MeshParty#80]
Using `meshparty.trimesh_io._download_meshes_thread_precomputed` fails when the source is a cloudvolume `ShardedMultiLevelPrecomputedMeshSource` since that source's `get` method does not take a `remove_duplicate_vertices` kwarg. 

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<env>/lib/python3.9/site-packages/meshparty/trimesh_io.py", line 458, in download_meshes
    mu.multiprocess_func(_download_meshes_thread,
  File "<env>/lib/python3.9/site-packages/multiwrapper/multiprocessing_utils.py", line 64, in multiprocess_func
    result.append(func(p))
  File "<env>/lib/python3.9/site-packages/meshparty/trimesh_io.py", line 366, in _download_meshes_thread_precomputed
    cv_meshes = cv.mesh.get(
TypeError: get() got an unexpected keyword argument 'remove_duplicate_vertices'
```

I'm not sure if this belongs as an issue in cloud-volume to make their API consistent across sources or here! A simple fix would be a try-except with a warning that the kwarg is being dropped.

**Comment by jboulanger91** [comment]
I am facing the same issue. Has anyone found a workaround? 
Thanks a lot, 
J

**Comment by fcollman** [comment]
I pushed a fix to this just now, 1.16.12 should now have this fix.
